securedrop-proxy/debian/rules

```diff
@@ -3,7 +3,7 @@
 %:
 	dh $@ --with python-virtualenv
 
-REQUIREMENTS_FILE=requirements/build-requirements.txt
+REQUIREMENTS_FILE=build-requirements.txt
```

Comment:
What do you think of something like:
```
REQUIREMENTS_FILE=$(shell test -e build-requirements.txt && echo "build-requirements.txt" || echo "requirements/build-requirements.txt")
```

That should support both the old and new locations. I would much prefer being able to rely on CI passing at every stage of this if possible...

Reply:
Yep, that works well. I'd prefer to revert that pretty quickly so we don't forget about it, will get a PR ready to do so.